app/src/main/java/com/andryoga/safebox/ui/view/MainActivity.kt

```diff
@@ -55,8 +64,11 @@ class MainActivity : AppCompatActivity() {
     object Constants {
         const val LAST_INTERACTED_TIME = "last_interacted_time"
         const val MAX_USER_AWAY_MILLI_SECONDS = 20000L
+        const val AUTO_UPDATE_REQUEST_CODE = 11
     }
 
+    private lateinit var appUpdateManager: AppUpdateManager
```

Comment:
shift this line to line 48 to keep all lateinit var together, it's easy to read code this way